capstone/src/main/java/capstone/Extras/Helper.java

```diff
@@ -623,6 +643,114 @@ public static void printUserCredentials(
     System.out.println(table);
   }
 
+  public static void printUserCredentials_customer(
```

Comment:
I think there is a mis-communication here. No user should have the password field in the table. not just customers. I think you can just remove the password line in the original function rather than rewriting a new one.